Makefile

```diff
@@ -35,7 +35,7 @@ endif
 
 .PHONY: test
 test:
-	go test -race $(PKGS)
+	go test -p 1 -race $(PKGS)
```

Comment:
what does this do and why is this needed?

Reply:
We don't want multiple suites to run in parallel since we have a test to restart zookeeper.

Reply:
is it possible to blacklist parallel run for that particular test instead?

Reply:
No tests in a suite are parallel by default. https://github.com/stretchr/testify/issues/52